darts/models/forecasting_model.py

```diff
@@ -360,9 +372,13 @@ def backtest(self,
                                               verbose)
 
         if last_points_only:
-            return metric(series, forecasts)
+            return metric(series[start:][forecast_horizon-1:], forecasts)
```

Comment:
why this change?

Reply:
I think I changed this at some point because for me some tests were failing. I'm pretty sure now this was a fault of my new implementation.

I could change it back because it does not change the result, but now the first argument `series[start:][forecast_horizon-1:]` has the same length and time index as the second `forecasts` argument. Before I changed it the `series` had another time index than the forecasts and they were intersected in the `metric` itself. Let me know if I should change it back.

Reply:
I think you could change it back. If we are confident the intersection is as expected, then I would find it clearer to rely on `metric` to do the intersection.